wpengine-geoip.php

```diff
@@ -578,6 +578,53 @@ private function match_label_synonyms( $label ) {
 
 		return $label;
 	}
+
+	/**
+	 * Utility function: Calculate distance to point
+	 *
+	 * Provided a lat/lng, calculate the distance from visitor's location
+	 * Uses the Haversine Formula, accurate for short distance but not over poles or the equator
+	 *
+	 * Note: Test against a return value of false to make sure you got a calculated distance. Example:
+	 	$geo = WPEngine\GeoIp::instance();
+        if ( false !== $geo->distance_to( $latitude, $longitude ) ) {
+			// Do something
+        }
+     *
+     * @link http://andrew.hedges.name/experiments/haversine/
+	 * @param float Latitude of the destination in degrees
+	 * @param float Longitude of the destination in degrees
+	 * @param bool  Whether to calculate the distance in kilometers or miles
+	 * @return float distance in miles
+	 * @since 1.2
+	 */
+	public function distance_to( $lat, $lng, $metric = false ) {
```

Comment:
Check out https://github.com/ryanshoover/geoip/blob/master/wpengine-geoip.php

If you look at line 589, something didn't merge correctly.  Code is just sitting there inside the class, outside any defined function.